bbl/i18n/OrcaSlicer.pot

```diff
@@ -18,6 +18,18 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
+msgid "Maximum speed of X axis"
```

Comment:
Thanks, @Guilouz 
This file is generated automatically by the `xgettext` application. Any manual changes made to this file will be overwritten when the xgettext application is executed in the future.

In case we want to localized more strings/messages, the suggested way is add `_L()` wrapper around the string to be translated in C++ source codes. e.g. `_L("message_to_translate")`